ImplementationGuide/markdown/Katalog/CodeSystem_Motivation.md

```diff
@@ -1,6 +1,12 @@
 ### Motivation
 
+Dieses Profil MUSS im Rahmen der Bestätigung NICHT unterstützt werden. Falls das Profil unterstützt werden soll, SOLLEN die hier definierten Festlegungen greifen (auch im Bestätigungsverfahren). 
+
 ISiK erlaubt in diversen Kontexten die Erweiterung der Kodierung durch Krankenhaus- / Systeminterne Kodierungen.
+
 Eine maschinenlesbare Repräsentation dieser Kodierungen erlaubt es Clients dazugehörige Anzeigetext und Definitionen zu verarbeiten.
 
+Da ISiKValueSet expandierte Valuesets vorsieht, ist eine dynamische Expansion in der Regel nicht erforderlich.
```

Comment:
nitpick (non-blocking): ich nehme an, der Absatz begründet warum das Profil auf optional gesetzt wurde. Einem Leser ist allerdings die Historie und der Entscheidungsprozess nicht bekannt, sodass der Absatz gefühlt kontextlos rüberkommt. Vllt. sollten wir den Motivationsabschnitt in 2 logische Teile trennen - 1. was sind die CodeSysteme und wozu werden sie im ISIK-Kontext verwendet und 2. Bestätigungsrelevanz (ggf. mit Begründung)

Reply:
@alexzautke schaust du dir nochmal an, ob der Absatz passt?